src/components/Carousel.js

```diff
@@ -0,0 +1,37 @@
+import { Component } from "react"
+import styled from "styled-components"
+
+const _FlickityContainer = styled.div``
+
+class Carousel extends Component {
+  state = { Flickity: null }
+  constructor(props) {
+    super(props)
+    if (typeof window !== "undefined") {
+      this.state.Flickity = require("react-flickity-component")
+    }
+  }
+  render() {
+    const { Flickity } = this.state
+    const { children } = this.props
+    return (
+      <_FlickityContainer>
+        {Flickity && (
+          <Flickity
+            options={{
+              prevNextButtons: false,
```

Comment:
@abdhalees We want the wrap around effect so I think the options that we want are  -

```
{ prevNextButtons: false
, wrapAround: true
}
```
We also don't want `freeScroll: true` as we want the cards to slot into position as you scroll.